### ISSUE TITLE
Tree view - contradiction re:selection following focus

### ISSUE THREAD

**Issue by kbae00** [w3c/aria-practices#652]
Near the beginning of the Tree view article, the APG says "If the tree allows the user to choose just one item for an action, then it is known as a single-select tree, and the item with focus also has a selected state. However, in multi-select trees, which enable the user to select more than one item for an action, the selected state is independent of the focus." I read that as saying that if it's a single-select tree, selection should follow focus.

However, in the Notes section, selection-follows-focus seems to be optional:
- Note #1: "DOM focus (the active element) is functionally distinct from the selected state"
- Note #3: "In a single-select tree, moving focus may optionally unselect the previously selected node and select the newly focused node."

This seems like a contradiction or at least it's not 100% clear.

**Comment by 0ddfell0w** [comment]
> the APG says "If the tree allows the user to choose just one item for an action, then it is known as a single-select tree, and the item with focus also has a selected state. However, in multi-select trees, which enable the user to select more than one item for an action, the selected state is independent of the focus." I read that as saying that if it's a single-select tree, selection should follow focus.

I don't think you should read that as saying a single-select tree should always have selection following focus. It's just pointing out that it's impossible for selection to follow focus in a multi-select tree. In a single select tree, it's a choice

**Comment by katiehockman** [comment]
I agree kbae00 that it is a bit misleading. The fact that the APG says, in the scope of a single-select tree, "the item with focus also has a selected state" that is describing selection-follows-focus. Maybe rewording it would be enough. Saying something like this might be clearer:
"In multi-select trees, which enable the user to select more than one item for an action, the selected state is independent of the focus. However, if the tree allows the user to choose just one item for an action, then it is known as a single-select tree, and selection-follow-focus may be implemented such that the item with focus also has a selected state."

**Comment by mcking65** [comment]
@0ddfell0w is right, but I agree with @kbae00 that the language is not consistent.

So, I pushed commit c7860cc.

If this does not address the issue fully, please feel free to re-open.
